spec/api.json

```diff
@@ -34,7 +34,7 @@
         "type": "string"
       },
       "Upstream": {
-        "description": "Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream upstream",
+        "description": "Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream Upstream upstream",
```

Comment:
:shrug:

Reply:
What you have to do to resolve this kind of bug is remove the generated SDK before you run swagger:
https://github.com/ory/keto/blob/d007bae4ce7b507f17f5c907bf90f858cfcd2950/Makefile#L70-L75